tests/e2e/bin/scripts/parallel/pages.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && cd ../../pages && pwd)"
+
+wp post create \
+	--menu_order=0 \
+	--post_type=page \
+	--post_status=publish \
+	--post_author=1 \
+	--post_title='All Products block' \
+	${script_dir}/all-products.html
```

Comment:
Is it still necessary?

Reply:
I'm not sure. In this PR, I'm trying to convert the old script to the new one only. Searching our tests, look like we aren't using that page anywhere.

Reply:
@gigitux and @dinhtungdu In https://github.com/nielslange/woo-test-environment, the `All Products block` is used for the shop page, see https://github.com/nielslange/woo-test-environment/blob/5cfa23af4a4aa426f71b7a626ad8a3786f722ccb/command.php#L212-L216

I'll try to review this PR later today and will double-check if the `All Products block` is needed or not.

Reply:
@gigitux and @dinhtungdu These are the only places, where we interact with the `All Products block`:

https://github.com/woocommerce/woocommerce-blocks/blob/1d0af208720498af118e76f9e0e3c641f708b1e0/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts#L31 

https://github.com/woocommerce/woocommerce-blocks/blob/1d0af208720498af118e76f9e0e3c641f708b1e0/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts#L46-L48

https://github.com/woocommerce/woocommerce-blocks/blob/1d0af208720498af118e76f9e0e3c641f708b1e0/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts#L77-L79

https://github.com/woocommerce/woocommerce-blocks/blob/1d0af208720498af118e76f9e0e3c641f708b1e0/tests/e2e/tests/mini-cart/mini-cart.block_theme.spec.ts#L113

As far as I can tell, the page `All Products block` is not loaded directly anywhere and can safely be removed.

Reply:
I removed the All Products block page in https://github.com/woocommerce/woocommerce-blocks/pull/10611/commits/4473ffcba170d7363b293280ed1254cde7ef7fc0